FPShellSortTord/src/js/Shellsort.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
```

Comment:
The naming and structure is a bit hard to understand. Do not mutate input parameters.